### PR TITLE
Enh #47: Implementation of the "Invite new users" endpoint

### PR DESCRIPTION
**Pull request by marc-farre**
Issue https://github.com/humhub/rest/issues/47

### CODE REVIEW

File: docs/swagger/user.yaml [hunk 1]
```diff
@@ -465,9 +465,39 @@ paths:
...
+          name: body
+          description: Email of people you want to invite
+          required: true
+          schema:
+            type: object
+            required:
+              - email
+            properties:
+              email:
+                type: string
```
Comment (luke-) [inline docs/swagger/user.yaml:491]: Can we use an array here? This probably makes automated things easier .

**Review by luke- — CHANGES_REQUESTED** [review]
@marc-farre Thanks, please see my remarks.

File: docs/swagger/user.yaml [hunk 2]
```diff
@@ -465,9 +465,39 @@ paths:
...
-# Begin Session
-#-----------------------------------------------------------------------------------------------------------------------
+  #-----------------------------------------------------------------------------------------------------------------------
+  # Begin Invite
+  #-----------------------------------------------------------------------------------------------------------------------
+  '/user/invite':
+    post:
+      tags:
+        - Invite
+      summary: Invite new users to the HumHub installation
```
[the remark this answers is not in the document]
Comment (marc-farre) [reply docs/swagger/user.yaml]: Done in commit https://github.com/humhub/rest/pull/138/commits/a6dc8a30d4d02691daedfa64d3ef3488b33af381

File: docs/swagger/user.yaml (hunk 1 above)
Comment (marc-farre) [reply docs/swagger/user.yaml]: Yes, array is better. I had initially chosen the comma separated string to use the Invite model.
Changed in commit https://github.com/humhub/rest/pull/138/commits/a6dc8a30d4d02691daedfa64d3ef3488b33af381

**Review by luke- — APPROVED** [review]

**Pull request merged**